### DISCUSSION TITLE
Getting SSLError when calling ProductFees get_product_fees_estimate_for_asin

### DISCUSSION THREAD

**Discussion by osvelqbano** [saleweaver/python-amazon-sp-api#801] [Q&A]
When trying to use the ProducFees endpoint I'm getting the following error. 

SSLError: HTTPSConnectionPool(host='api.amazon.com', port=443): Max retries exceeded with url: /auth/o2/token (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))

This is the code that I have:
-------------------------------------------------------------------------------------------------

from sp_api.api import ProductFees

credentials = properly_set_up -- Tested it with other endpoints 

fee =  ProductFees(credentials=credentials)
fee_json = fee.get_product_fees_estimate_for_asin(asin ='B09V6YXXH4', price = '29.99')
print(fee_json)

--------------------------------------------------------------------------------------------------
If I print(fee) I do not get said error. It only happens when I tackle on the .get_product_fees_estimate_for_asin()

Is anyone having the same issue ?

**Comment by jcabourne** [comment]
Hi 

Did you ever resolve this issue i am having a similar problem?
